tools/modules/software/module_hastebin.sh

```diff
@@ -32,18 +32,21 @@ function module_hastebin () {
 			pkg_installed docker-ce || module_docker install
 			[[ -d "$HASTEBIN_BASE" ]] || mkdir -p "$HASTEBIN_BASE" || { echo "Couldn't create storage directory: $HASTEBIN_BASE"; exit 1; }
 			mkdir -p "$HASTEBIN_BASE/pastes"
-			chown -R 1000:1000 "$HASTEBIN_BASE" # this is necessary as the container runs as user 1000 (node)
+
+			wget -qO- https://raw.githubusercontent.com/armbian/hastebin-ansi/refs/heads/main/about.md > "$HASTEBIN_BASE/about.md"
```

Comment:
Perhaps just creating file with (web) URL?

Reply:
> Perhaps just creating file with (web) URL?

I couldn't understand. Can you clarify it

Reply:
Creating a file README.md with a content:
https://github.com/armbian/hastebin-ansi/blob/main/about.md